docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,56 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+The Government of the Netherlands demands that government organisations are transparent about the use of
+high impact algorithms. This is a requirement of
+the [Open Government Act](https://www.rijksoverheid.nl/onderwerpen/wet-open-overheid-woo).
+This transparency can start with registering an algorithm in the
+[Algoritmeregister](https://algoritmes.overheid.nl/).
+
+Validation can be achieved by proper evaluation of algorithmic performance,
+bias and fairness and by doing assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+Transparency can be achieved by sharing these outcomes in a standardized way.
+
+There are several existing open source
+[tools](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools/) which
+we [have assessed](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](https://minbzk.github.io/ai-validation/Projects/TAD/Tools/) we have.
+
+These are our main requirements:
+
+* The tool offers A one-stop shop for all aspects of the project for all stakeholders for all tasks.
+* The tool supports a workflow to track different stages of the project.
+* The tool can run different technical tests on a model.
+* The tool supports filling out assesments.
+* The tool supports discussion features.
+* The tool offers UI access to the above requirements.
+* The tool can save results to a system card (or cards supported by system cards, like model, metrics and assessment).
+* The tool can push the different cards to a VCS like GitHub to allow for an audit trail.
+
+## Assumptions
+
+* Collaborating or extending another project will not give us the tool we are looking for.
+* We can reuse certain components, like the
+[plugins from AIVerify](https://github.com/IMDA-BTG/aiverify/tree/main/stock-plugins) for technical tests.
+* The tool will use a design based on loose coupled modules. This can be done by scanning directories,
+  to gather modules that implement a certain Interface.
+* Plugins will have to implement an Interface to be picked up by the system.
+* We can, to some extend, re-use the already existing POCs or findings from these POCs.
+
+## Decision
+
+We will develop our own tool. Where possible this tool should be able to re-use certain components of other open-source
+tools.
+
+## Risks
+
+* It is a lot of work to create our own tool.
+* We may not have all the knowledge yet to create technical tests.
```

Comment:
I don't think we should create any technical test, we should reuse existing tests (we already mention this in our assumptions)

Reply:
I believe a bigger risk is that we may struggle to get uptake of the tool if we are not aligned with envisioned users of the tool